src/schemas/json/cloudinit.json

```diff
@@ -0,0 +1,390 @@
+            }
+        },
+        {
+            "id": "cc_zypper_add_repo",
+            "name": "ZypperAddRepo",
+            "title": "Configure zypper behavior and add zypper repositories",
+            "description": "Configure zypper behavior by modifying /etc/zypp/zypp.conf. The\nconfiguration writer is \"dumb\" and will simply append the provided\nconfiguration options to the configuration file. Option settings\nthat may be duplicate will be resolved by the way the zypp.conf file\nis parsed. The file is in INI format.\nAdd repositories to the system. No validation is performed on the\nrepository file entries, it is assumed the user is familiar with\nthe zypper repository file format.",
+            "distros": [
+                "opensuse",
+                "sles"
+            ],
+            "examples": [
+                "zypper:\n  repos:\n    - id: opensuse-oss\n      name: os-oss\n      baseurl: http://dl.opensuse.org/dist/leap/v/repo/oss/\n      enabled: 1\n      autorefresh: 1\n    - id: opensuse-oss-update\n      name: os-oss-up\n      baseurl: http://dl.opensuse.org/dist/leap/v/update\n      # any setting per\n      # https://en.opensuse.org/openSUSE:Standards_RepoInfo\n      # enable and autorefresh are on by default\n  config:\n    reposdir: /etc/zypp/repos.dir\n    servicesdir: /etc/zypp/services.d\n    download.use_deltarpm: true\n    # any setting in /etc/zypp/zypp.conf\n"
+            ],
+            "frequency": "always",
+            "type": "object",
+            "properties": {
+                "zypper": {
+                    "type": "object",
+                    "properties": {
+                        "repos": {
+                            "type": "array",
+                            "items": {
+                                "type": "object",
+                                "properties": {
+                                    "id": {
+                                        "type": "string",
+                                        "description": "The unique id of the repo, used when\n writing\n/etc/zypp/repos.d/<id>.repo."
+                                    },
+                                    "baseurl": {
+                                        "type": "string",
+                                        "format": "uri",
+                                        "description": "The base repositoy URL"
+                                    }
+                                },
+                                "required": [
+                                    "id",
+                                    "baseurl"
+                                ],
+                                "additionalProperties": true
+                            },
+                            "minItems": 1
+                        },
+                        "config": {
+                            "type": "object",
+                            "description": "Any supported zypo.conf key is written to\n/etc/zypp/zypp.conf'"
```

Comment:
```suggestion
                            "description": "Any supported zypo.conf key is written to\n'/etc/zypp/zypp.conf'"
```

Reply:
You should submit a suggestion to the original file: https://github.com/canonical/cloud-init/blob/master/cloudinit/config/cc_zypper_add_repo.py